libs/inspecJS/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "inspecjs",
+  "version": "2.4.6",
```

Comment:
Is jumping from version 0.12.4 to 2.4.6 going to cause issues? If we are going to match InspecJS with Heimdall can this be updated to the current release? (2.4.8)

Reply:
I don't believe this will cause any issues, so I also think it should be fine to update it to the current release.

Reply:
Lerna automatically modifies these on release as well.